VirtoCommerce.Domain/Customer/Model/Organization.cs

```diff
@@ -1,11 +1,50 @@
-﻿namespace VirtoCommerce.Domain.Customer.Model
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using VirtoCommerce.Domain.Commerce.Model;
+using VirtoCommerce.Platform.Core.DynamicProperties;
+
+namespace VirtoCommerce.Domain.Customer.Model
 {
-    public class Organization : Member
+    public class Organization : Member, ICloneable
     {
         public string Description { get; set; }
         public string BusinessCategory { get; set; }
         public string OwnerId { get; set; }
         public string ParentId { get; set; }
 
+        public virtual object Clone()
+        {
+            var clone = (Organization) MemberwiseClone();
+
+            clone.Addresses = Addresses
```

Comment:
I think we need to add support ICloneable to Member (and probably to all its descendants). MemberwiseClone and cloning its collection fields should be done inside base Member class, descendants should call base implementation + add own info.

Reply:
I completely agree. See commit 4a76c3ff26a45968f1b4c3aa9acd1b3332f461b0.